src/main/java/com/chessmates/repository/MetaDataRepository.groovy

```diff
@@ -1,27 +1,24 @@
 package com.chessmates.repository
 
-import com.chessmates.model.Game
-import com.chessmates.model.Player
 import com.google.common.collect.ImmutableMap
-import org.apache.commons.lang3.tuple.ImmutablePair
 
 /**
  * A service for storing & accessing information about the current state of Lichess data.
  */
 interface MetaDataRepository {
 
     /** Save latest player fetched from the Lichess API. */
-    void saveLatestPlayer(Player player)
+    void saveLatestPlayer(player)
 
     /** Get latest player fetched from the Lichess API. */
-    Player getLatestPlayer()
+    def getLatestPlayer()
 
     /** Save the latest game requested from the Lichess API for a given set of opponents. */
-    void saveLatestGame(Player player, Player opponent, Game game)
+    void saveLatestGame(player, opponent, game)
 
     /** Get the latest game request from the Lichess API for a given set of opponents.
      * ImmutableMap is returned to indicate that changes to this store don't affect the store contents.
      */
-    ImmutableMap<ImmutablePair<Player, Player>, Game> getLatestGames()
+    ImmutableMap getLatestGames()
```

Comment:
I don't think we had to lose ImmutablePair to Object mapping here. I think keeping it is a good idea in the signature.